app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("The visitor should have a 'vaccine' key")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("The 'vaccine' key is out-dated")

```

Comment:
it's better to write messages that are understandable for users: `f"{visitor['name']} has outdated vaccine"` for example. Key can not be outdated

Reply:
Fix it everywhere